package.json

```diff
@@ -23,22 +23,26 @@
     "@wordpress/data": "^10.11.0",
     "@wordpress/element": "^6.11.0",
     "@wordpress/html-entities": "^4.11.0",
+    "glob": "^11.0.0",
     "@wordpress/scripts": "^24.6.0",
     "babel-loader": "^8.2.5",
-    "css-loader": "^6.11.0",
-    "glob": "^11.0.0",
+    "css-loader": "^6.7.1",
     "node-sass": "^7.0.3",
     "react": "^18.3.1",
+    "grunt-contrib-cssmin": "^5.0.0",
+    "grunt-contrib-uglify": "^5.2.2",
+    "grunt-wp-i18n": "^1.0.3",
     "sass": "^1.55.0",
-    "sass-loader": "^13.3.3",
+    "sass-loader": "^13.1.0",
     "style-loader": "^4.0.0",
     "ts-loader": "^9.5.1",
     "typescript": "^5.6.3",
-    "webpack": "^5.96.1",
+    "webpack": "^5.74.0",
     "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.15.2"
+    "webpack-dev-server": "^4.11.1"
   },
   "scripts": {
+    "prebuild": "npx grunt makepot && npx grunt cssmin",
```

Comment:
This causes it to no longer build for me. I think we need to do two changes. Add grunt-cli as a dev dependency with `pnpm i --save-dev grunt-cli` and change the filetype of the `gruntfile.js` to be `gruntfile.cjs` to get it to build properly.